### PR TITLE
Unit testing for stock service

### PR DESCRIPTION
**Pull request by murradkh**

### CODE REVIEW

File: myapp/tests/example.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+from django.test import TestCase, Client
```
Comment (kamalq97) [inline myapp/tests/example.py:1]: Note that in order to be "seen" by Django as test file. It needs to have the name "test_*.py" (I imagine you probably already know this...?)

**Review by kamalq97 — APPROVED** [review]
Overall, this is a good test case that covers all the basic scenarios well and tests for somethings that were difficult to do with manual testing. 

One minor thing: I am not sure why you defined a full list of existent symbols but only seem to use the first element in the list sometimes and at other times, you loop over all the elements in the list. I am guessing this is just an arbitrary decision that you made?  Anyhow, it doesn't really matter all that much, so you may keep it as is.

File: myapp/tests/example.py (hunk 1 above)
Comment (murradkh) [reply myapp/tests/example.py:1]: yes, due to it's an `example` test (for us to see how the content would look like), I removed the prefix to not running in the actual test; on the other side, as your suggestion, to keep the prefix in order to see how test files named
there is a tradeoff here :)

**Comment by murradkh** [comment]
> Overall, this is a good test case that covers all the basic scenarios well and tests for somethings that were difficult to do with manual testing.
> 
> One minor thing: I am not sure why you defined a full list of existent symbols but only seem to use the first element in the list sometimes and at other times, you loop over all the elements in the list. I am guessing this is just an arbitrary decision that you made? Anyhow, it doesn't really matter all that much, so you may keep it as is.

thanks, 
when looping over the elements it testing different inputs,  while when taking the first element is intended to test other cases (other than a different-inputs case)

**Pull request merged**